tools/windows-node-installer/go.mod

```diff
@@ -17,3 +19,9 @@ require (
 	k8s.io/utils v0.0.0-20190809000727-6c36bc71fc4a // indirect
 	sigs.k8s.io/controller-runtime v0.2.0
 )
+
+replace (
+	k8s.io/api => k8s.io/api v0.0.0-20190620084959-7cf5895f2711 // kubernetes-1.15.0
```

Comment:
I am not sure if this will be used with 4.2 for the drop, so we might want to stick to kube v1.14

Reply:
Was the comment manually added?

Reply:
Yes, the comment about the version name is manually added. Since go build always changes the version name to time and commit number, I think the comment would help and will not be erased.